lib/Service/ApiService.php

```diff
@@ -113,26 +112,15 @@ public function create($fileId = null, $filePath = null, $token = null, $guestNa
 			$readOnly = $this->documentService->isReadOnly($file, $token);
 
 			$this->sessionService->removeInactiveSessionsWithoutSteps($file->getId());
-			$remainingSessions = $this->sessionService->getAllSessions($file->getId());
-			$freshSession = false;
-			if ($forceRecreate || count($remainingSessions) === 0) {
```

Comment:
Seems like `$forceRecreate` is no longer used anywhere. Is there no need for it anymore? Then I'd also remove it from the function parameters.